scene/3d/sprite_3d.cpp

```diff
@@ -1246,37 +1244,33 @@ void AnimatedSprite3D::play(const StringName &p_animation, bool p_backwards) {
 
 void AnimatedSprite3D::stop() {
 	set_playing(false);
+	backwards = false;
```

Comment:
`backwards` is actually overwritten on the next play, so initializing may not be necessary, but it is not particularly odd, so I would consider it not a blocker.

Reply:
Animation can be started not via `play()`, but via `playing = true`.

Reply:
If so, it seems to me that it would be better to retain the previous backward. But perhaps we need to rethink this again when we focus on pause.

Reply:
Well, it seems a bit odd that there are two ways to playback.
For now, if you respect your code, it might be better to document that `stop()` acts as a reset direction to forward and the direction of playback is kept for `is_playing = false/true`.

Reply:
Done. @YuriSizov Could you please check the doc changes in this PR?

Reply:
Since the `backward` parameter is not public, I think it needs to note that it is an internal state.

Reply:
I meant the method parameter, I think the user will understand that the parameter affects the internal state of the object. Or perhaps somehow it is possible to reformulate this, removing the `backwards` mention?

Reply:
To avoid users searching for the `backwards` parameter, I prefer an ambiguous sentence such as "The direction of playback as stored internally will be kept".

Reply:
"playback direction" may confuse the user that the `stop` method also resets negative `speed_scale`. It seems to me that the reference to the `backwards` parameter and the `play` method is necessary.

Reply:
Well, as long as there is a proper reference to the play() method, it may be okay. I leave it to @akien-mga or @YuriSizov to make the final decision.

Reply:
Checked the docs 👍